src/main.ts

```diff
@@ -48,11 +48,11 @@ export default async function run() {
 
 function getInputs(): IActionInputs {
     core.debug('Get action inputs.');
-    let serverName = core.getInput('server-name', { required: true });
```

Comment:
Here also, let's not remove this
We can have it both ways : if the user has provided the server name, that can be considered, otherwise it can be parsed from the connection string.

This will ensure that the existing flow doesn't break.

Reply:
yeah, make sense. I'll do it ASAP

Reply:
If the use specify both the server name in the server-name parameter and in the connection-string which one should take precedence? I would say the connection-string. What do you think?

Reply:
Since the user is explicitly mentioning the server name, let's give that a preference.
If there's no server name, we can extract it from the connection string.

Reply:
If there is a disparity, let's log that to the console output for the user - regardless of which takes precedence.